### PR TITLE
Formatting, Typos

### PR DESCRIPTION
**Pull request by OG-NI**
- fixed formatting, typos
- added prettier config for docs
- added npm scripts and github action to check formatting of docs for each PR

### CODE REVIEW

**Comment by OG-NI** [comment]
@benedictweis please check if the github action is correct

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    runs-on: [self-hosted]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Run check format
+        run: npm run check:format
```
Comment (benedictweis) [inline .github/workflows/docs.yml:19]: Does it need to install any dev dependencies, if so, add a step that installs the dependencies.

File: .github/workflows/docs.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+jobs:
+  verify:
+    runs-on: [self-hosted]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
```
Comment (benedictweis) [inline .github/workflows/docs.yml:17]: Node needs to be installed first, see `backend.yaml`.

File: .github/workflows/docs.yml (hunk 1 above)
Comment (benedictweis) [inline .github/workflows/docs.yml:19]:
```suggestion
        run: cd docs && npm run check:format
```

You need to cd into `docs` before running any command.

**Review by benedictweis — CHANGES_REQUESTED** [review]
Thanks for the fixes, the PR looks great. I have added some things regarding the action, please see my comments.

**Review by benedictweis — APPROVED** [review]
The action looks good, lets merge and find out if it works in the next PR 😉.

**Pull request merged**